<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Venue_Label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Venue:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Date_Label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date &amp; Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Venue_Entry">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="Series_RadioButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Series</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Tournament_RadioButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tournament</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Stage_ComboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Qualifiers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Quarter-Finals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Semi-Finals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Finals</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Stage_Label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stage:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Tournament_ComboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Tournament_Label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tournament:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Team_1_Label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team 1:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Team_1_ComboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>150</y>
     <width>21</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Match_Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Match</string>
   </property>
  </widget>
  <widget class="QLabel" name="Series_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Series: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="Series_ComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Series_Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Series</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Tournament_Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Tournament</string>
   </property>
  </widget>
  <widget class="QLabel" name="Team_2_Labe">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team 2:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Team_2_ComboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Date_Time_Entry">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f00;</string>
   </property>
   <property name="text">
    <string>Please provide all details</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
